src/dfx/assets/project_templates/vanilla_js/src/__frontend_name__/assets/.ic-assets.json5

```diff
@@ -1,19 +1,20 @@
 [
-    {
-        "match": "**/*",
-
-        // Provides a base set of security headers that will work for most dapps.
-        // Any headers you manually specify will override the headers provided by the policy.
-        // See 'dfx info security-policy' to see the policy and for advice on how to harden the headers.
-        // Once you improved the headers for your dapp, set the security policy to "hardened" to disable the warning.
-        // Options are: "hardened" | "standard" | "disabled".
-        "security_policy": "standard",
-
-        // Uncomment to disable the warning about using the
-        // standard security policy, if you understand the risk
-        // "disable_security_policy_warning": true,
-
-        // Uncomment to redirect all requests from .raw.icp0.io to .icp0.io
-        // "allow_raw_access": false
+  {
+    "match": "**/*",
+    // Provides a base set of security headers that will work for most dapps.
+    // Any headers you manually specify will override the headers provided by the policy.
+    // See 'dfx info security-policy' to see the policy and for advice on how to harden the headers.
+    // Once you improved the headers for your dapp, set the security policy to "hardened" to disable the warning.
+    // Options are: "hardened" | "standard" | "disabled".
+    "security_policy": "standard",
+    "headers": {
+      "Content-Security-Policy": "default-src 'self';script-src 'self' 'unsafe-eval' 'unsafe-inline';connect-src 'self' http://localhost:* https://icp0.io https://*.icp0.io https://icp-api.io;img-src 'self' data:;style-src * 'unsafe-inline';style-src-elem * 'unsafe-inline';font-src *;object-src 'none';base-uri 'self';frame-ancestors 'none';form-action 'self';upgrade-insecure-requests;",
```

Comment:
Can you explain the rationale for overriding the standard security policy's Content-Security-Policy here, rather than updating the header for policy at https://github.com/dfinity/sdk/blob/master/src/canisters/frontend/ic-asset/src/security_policy.rs#L95 ?

The sveltekit starter overrides the standard security policy's Content-Security-Policy header "due to the use of inline scripts" (see https://github.com/dfinity/sdk/pull/3902, merged to the release-0.23.0 branch and later merged to main when promoting that release).  Unless there's a good reason, we shouldn't copy that to the other frontend templates.